roles/prometheus/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2024, Itential, Inc
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Install base OS packages
+  ansible.builtin.include_role:
+    name: os
+  tags: install_base_os_packages
+
+- name: Create prometheus group
+  ansible.builtin.group:
+    name: "{{ prometheus_group }}"
+    state: present
+
+- name: Create prometheus user
+  ansible.builtin.user:
+    name: "{{ prometheus_user }}"
+    group: "{{ prometheus_group }}"
+    state: present
+
+- name: Create a directory if it does not exist
+  ansible.builtin.file:
+    path: /opt/prometheus
+    state: directory
+    owner: "{{ prometheus_user }}"
+    group: "{{ prometheus_group }}"
+    mode: "0755"
+
+# Install prometheus on its on server
+- name: Install Prometheus server
+  tags: prometheus_server
+  ansible.builtin.import_tasks:
+    file: install_prometheus.yml
+  when: inventory_hostname in groups['prometheus']
+
+# Install the node exporter on all servers
+- name: Install node exporter on all hosts
+  tags: node_exporter
+  ansible.builtin.import_tasks:
+    file: install_node_exporter.yml
+  when: inventory_hostname not in groups['prometheus']
+
+# Install the process exporter on platform servers
```

Comment:
Maybe we should rename the "process_exporter" to "iap_process_exporter"?  I was confused as to why we were only installing the process_exporter on the platform servers.
Also, do we need process exporters for the other components?  Or do they automatically contain process exporters?